src/server/qgsfcgiserverresponse.h

```diff
@@ -45,11 +45,13 @@ class QgsSocketMonitoringThread: public QThread
      * \param  isResponseFinished
      * \param  feedback
      */
-    QgsSocketMonitoringThread( bool *isResponseFinished, QgsFeedback *feedback );
+    QgsSocketMonitoringThread( QgsFeedback *feedback );
```

Comment:
What about the feedback? Should it become a shared ptr now that lifetime is decoupled?

Reply:
I need to check the side effects

Reply:
done!